racket/collects/racket/hash.rkt

```diff
@@ -1,5 +1,6 @@
 #lang racket/base
-(require racket/contract/base)
+(require racket/contract/base
+         (only-in racket/set set-intersect))
```

Comment:
I don't know how matter this is gonna be, but this change increases the startup time of the program:

```
#lang racket/base
(require racket/hash)
```

from 0.18s to 0.22s. If it matters, you could use a (mutable) hash mapping to `#t` to simulate a set to avoid requiring `racket/set`.

Reply:
Thank you @sorawee for your suggestion.  I ended up doing the intersection using a `for/fold` with a list accumulator, so no `set-intersect` any more.